cluster-operator/src/test/java/io/strimzi/operator/cluster/model/CruiseControlTest.java

```diff
@@ -276,10 +270,6 @@ public void testBrokerCapacities() {
 
         Kafka resource = createKafka(cruiseControlSpec);
 
-        Capacity capacity = new Capacity(Reconciliation.DUMMY_RECONCILIATION, resource.getSpec(), NODES, createStorageMap(resource), createResourceRequirementsMap(resource));
```

Comment:
I think you also need to get rid of line 271.

Reply:
Moved things around, let me know what you think